ledger/participant-state-index/src/main/scala/com/daml/ledger/participant/state/index/v2/IndexService.scala

```diff
@@ -3,13 +3,12 @@
 
 package com.daml.ledger.participant.state.index.v2
 
-//TODO: rename all subtraits to contain "Index" prefix in a follow-up PR!
 trait IndexService
-    extends PackagesService
-    with ConfigurationService
+    extends IndexPackagesService
+    with IndexConfigurationService
     with IndexCompletionsService
-    with TransactionsService
-    with ActiveContractsService
+    with IndexTransactionsService
+    with IndexActiveContractsService
     with ContractStore
-    with IdentityService
-    with TimeService
+    with IdentityProvider
+//with IndexTimeService //TODO: this needs some further discussion as the TimeService is actually optional
```

Comment:
i'd address this in a follow-up PR as we should discuss how we should handle it, TimeService being sort of an optional test service.